driver/src/price_finding/naive_solver.rs

```diff
@@ -110,11 +105,8 @@ impl PriceFinding for SnappNaiveSolver {
         let mut prices: Vec<u128> = vec![0; TOKENS as usize];
         let mut exec_buy_amount: Vec<u128> = vec![0; orders.len()];
         let mut exec_sell_amount: Vec<u128> = vec![0; orders.len()];
-        let mut total_objective_value = U256::zero();
-
-        let mut found_flag = false;
 
-        for (i, x) in orders.iter().enumerate() {
+        'outer: for (i, x) in orders.iter().enumerate() {
```

Comment:
`'outer: ` <---- What the heck is this? Some sort of space language?

Reply:
It a loop label so you can break from nested loops: https://doc.rust-lang.org/rust-by-example/flow_control/loop/nested.html